pkg/controllers/jobset_controller.go

```diff
@@ -371,54 +405,39 @@ func (r *JobSetReconciler) resumeJobsIfNecessary(ctx context.Context, js *jobset
 		}
 		jobsFromRJob := replicatedJobToActiveJobs[replicatedJob.Name]
 		for _, job := range jobsFromRJob {
+			if !jobSuspended(job) {
+				return nil
+			}
```

Comment:
By returning here, aren't we missing to resume the rest of the jobs? did you mean to use `continue`?

Reply:
Woops yes I meant to continue, thanks